### ISSUE TITLE
Fix CssSyntaxError regression for at-keyframes within mixin in Less

### ISSUE THREAD

**Issue by glutentag** [stylelint/stylelint#3771]
Since version 9.7.0 when using keyframes with mixin in Less, CssSyntaxError are thrown. The error is from postcss-less, I have already created [a ticket](https://github.com/shellscape/postcss-less/issues/122). This ticket is more a reminder to update postcss-less version once it is fixed.
> What CSS is needed to reproduce the bug?

```less
.mixin(@rules) {
    @keyframes loading-spin {
        @rules();
    }
}

.mixin({
    0% {
        transform: rotate(0deg);
    }
    100% {
        transform: rotate(360deg);
    }
});
```

> Which version of stylelint are you using?

9.7.1

> How are you running stylelint: CLI, PostCSS plugin, Node API?

stylelint test.less --syntax less

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

Yes, it's related to Less

> What did you expect to happen?

No errors to be thrown.

> What actually happened (e.g. what warnings or errors did you get)?

The following error was thrown:
```shell
test.less
 14:2  ✖  Unknown word   CssSyntaxError
```

**Comment by jwilsson** [comment]
This is fixed upstream in `postcss-less@3.1.0` so a fresh `npm install` should take care of it automatically.

**Comment by jwilsson** [comment]
With `9.9.0` the new version of `postcss-less` is included as well, so this is fixed.
